src/main/java/gregtech/api/pipenet/longdist/LongDistanceNetwork.java

```diff
@@ -174,6 +174,19 @@ protected void onDestroy() {
 
     public MetaTileEntityLongDistanceEndpoint getOtherEndpoint(MetaTileEntityLongDistanceEndpoint endpoint) {
         if (!isValid()) return null;
+        if (this.pipeType.getMinLength() > 0) {
+            for (int i = 0; i < this.endpoints.size(); i++) {
+                MetaTileEntityLongDistanceEndpoint other = this.endpoints.get(i);
+                if (endpoint != other && endpoint.getPos().getDistance(other.getPos().getX(), other.getPos().getY(), other.getPos().getZ()) >= this.pipeType.getMinLength()) {
```

Comment:
This distance calculation uses the distance formula to find an exact distance rather than the total count of pipeline blocks, which I think would be preferred

Reply:
I don't think it matters much

Reply:
I decided to not change it. It's either distance or nothing. Pipeline length would make everything so much more complex and i dont want to do that.